draft-ietf-httpbis-priority.md

```diff
@@ -324,37 +347,33 @@ transitions to a state that prevents the client from sending additional
 frames on the stream. Therefore, a client cannot reprioritize a response by
 using the Priority header field.  Modifying this behavior would require a
 semantic change to the protocol, but this is avoided by restricting the
-stream on which a PRIORITY_UPDATE frame can be sent. In HTTP/2 the frame
-is on stream zero and in HTTP/3 it is sent on the control stream
-({{!I-D.ietf-quic-http}}, Section 6.2.1).
-
-This document specifies a new PRIORITY_UPDATE frame type for HTTP/2
-({{!RFC7540}}) and HTTP/3 ({{!I-D.ietf-quic-http}}) which enables
-reprioritization. It carries updated priority parameters and references the
-target of the reprioritization based on a version-specific identifier; in
-HTTP/2 this is the Stream ID, in HTTP/3 this is either the Stream ID or Push ID.
-
-Unlike the header field, the reprioritization frame is a hop-by-hop signal.
+stream on which a PRIORITY_UPDATE frame can be sent.
```

Comment:
Should be more like

```
  Therefore, a client cannot reprioritize a response by
  sending additional metadata (e.g., trailer fields) on the request stream.
  We avoid interoperability failure by restricting PRIORITY_UPDATE
  frames to the control stream, with each such frame explicitly identifying
  the stream to which the new priority applies.
```

Reply:
Thanks. I tweaked the suggestion slightly and made cbcdc60cf5a92. PTAL.